apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/anyAssertions.kt

```diff
@@ -41,6 +41,19 @@ fun <T> Expect<T>.isSameAs(expected: T): Expect<T> = _logicAppend { isSameAs(exp
  */
 fun <T> Expect<T>.isNotSameAs(expected: T): Expect<T> = _logicAppend { isNotSameAs(expected) }
 
+/**
+ * Allows to state a reason for one or multiple assertions for a given subject.
+ * @param reason The explanation for the assertion/s.
```

Comment:
I’d add that we are talking about the assertions created by `assertionCreator`:

```suggestion
 * @param reason The explanation for the assertion(s) created by [assertionCreator].
```